packages/react-vis/package.json

```diff
@@ -80,6 +97,7 @@
     "react-addons-test-utils": ">=15.4.2",
     "react-dom": "^16.0.0",
     "react-test-renderer": "^16.13.1",
+    "react-vis-showcase": "^0.1.0",
```

Comment:
Hm why is this necessary? Won't it become a circular depedency?

Reply:
it is a ciruclar dependency.  But that dependency already existed since the tests directly reference the components in showcase.

We can fix this when we refactor the tests.
https://github.com/uber/react-vis/issues/1328